content/pt-br/docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -0,0 +1,713 @@
+sobre um mapa de configuração.
+
+#### Crie um mapa de configuração a partir de diretórios
+
+Você pode usar `kubectl create configmap` para criar um mapa de configuração a partir de vários arquivos no mesmo diretório. Quando você está criando um mapa de configuração baseado em um diretório, o kubectl identifica arquivos cujo nome-base é uma chave válida no diretório e empacota cada um desses arquivos no novo mapa de configuração. Quaisquer entradas de diretório, exceto que arquivos regulares são ignorados (ex. subdiretórios, links simbólicos, dispositivos, pipes, etc).
+
+Por exemplo:
+
+```shell
+# Criando o diretório local
+
+mkdir -p configure-pod-container/configmap/
+
+# Baixe os arquivos de amostra no diretório `configure-pod-container/configmap/` 
+
+wget https://kubernetes.io/examples/configmap/game.properties -O configure-pod-container/configmap/game.properties
+wget https://kubernetes.io/examples/configmap/ui.properties -O configure-pod-container/configmap/ui.properties
+
+# Crie o mapa de configuração
+
+kubectl create configmap game-config --from-file=configure-pod-container/configmap/
+```
+
+O comando acima empacota cada arquivo, neste caso, `game.properties` e `ui.properties` no diretório `configure-pod-container/configmap/` dentro do mapa de configuração de nome game-config. Você pode exibir detalhes do mapa de configuração usando o seguinte comando:
+
+```shell
+kubectl describe configmaps game-config
+```
+
+A saída é semelhante a esta:
+```
+Name:         game-config
+Namespace:    default
+Labels:       <none>
+Annotations:  <none>
+
+Data
+====
+game.properties:
+----
+enemies=aliens
+lives=3
+enemies.cheat=true
+enemies.cheat.level=noGoodRotten
+secret.code.passphrase=UUDDLRLRBABAS
+secret.code.allowed=true
+secret.code.lives=30
+ui.properties:
+----
+color.good=purple
+color.bad=yellow
+allow.textmode=true
+how.nice.to.look=fairlyNice
+```
+
+Os arquivos `game.properties` e `ui.properties` no diretório `configure-pod-container/configmap/` estão representados na seção `data` do mapa de configuração.
+
+```shell
+kubectl get configmaps game-config -o yaml
+```
+A saída é semelhante a esta:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  creationTimestamp: 2016-02-18T18:52:05Z
+  name: game-config
+  namespace: default
+  resourceVersion: "516"
+  uid: b4952dc3-d670-11e5-8cd0-68f728db1985
+data:
+  game.properties: |
+    enemies=aliens
+    lives=3
+    enemies.cheat=true
+    enemies.cheat.level=noGoodRotten
+    secret.code.passphrase=UUDDLRLRBABAS
+    secret.code.allowed=true
+    secret.code.lives=30
+  ui.properties: |
+    color.good=purple
+    color.bad=yellow
+    allow.textmode=true
+    how.nice.to.look=fairlyNice
+```
+
+#### Crie mapa de configuração a partir de arquivos
+
+Você pode usar `kubectl create configmap` para criar um mapa de configuração a partir de um arquivo individual, ou a partir de múltiplos arquivos.
+
+Por exemplo,
```

Comment:
```suggestion
#### Crie um ConfigMap a partir de arquivos {#create-configmaps-from-files}

Você pode usar `kubectl create configmap` para criar um ConfigMap a partir de um arquivo individual, ou a partir de múltiplos arquivos.

Por exemplo,
```

Here, the title also requires an anchor for links to work properly. Also preferable to not translate ConfigMap as it is an API object.